plone/content.md

```diff
@@ -64,25 +64,44 @@ The Plone image uses several environment variable that allow to specify a more s
 ### For Basic Usage
 
 -	`ADDONS` - Customize Plone via Plone add-ons using this environment variable
+-	`SITE` - Add Plone instance with this id to `Data.fs` on first run. If NOT provided, you'll have to manually add a Plone Site via web UI
 -	`ZEO_ADDRESS` - This environment variable allows you to run Plone image as a ZEO client.
+-	`VERSIONS` - Use specific versions of Plone Add-on or python libraries
 
-Run Plone with ZEO and install two addons (PloneFormGen and collective.roster)
+Run Plone and install two addons (eea.facetednavigation and collective.easyform)
 
 ```console
-$ docker run --name=instance1 --link=zeo -e ZEO_ADDRESS=zeo:8080 -p 8080:8080 \
--e ADDONS="Products.PloneFormGen collective.roster" plone
+$ docker run -p 8080:8080 -e SITE="mysite" -e ADDONS="eea.facetednavigation collective.easyform" plone
 ```
 
 To use specific add-ons versions:
 
 ```console
- -e ADDONS="Products.PloneFormGen==1.8.5 collective.roster==2.3.1"
+ -e ADDONS="eea.facetednavigation collective.easyform" -e VERSIONS="eea.facetednavigation=13.3 collective.easyform=2.1.0"
+```
+
+RestAPI:
+
+```console
+$ docker run -p 8080:8080 -e SITE=plone plone
```

Comment:
You may want `%%IMAGE%%` instead of plone so that it gets replace with `arch/plone` when viewed on and architecture specific namespace like https://hub.docker.com/r/arm64v8/plone/
```suggestion
$ docker run -p 8080:8080 -e SITE=plone %%IMAGE%%
```